src/components/DevDummyLogin/index.js

```diff
@@ -61,7 +61,7 @@ const S = StyleSheet.create({
 
 const DevDummyLogin = ({resetClickTime = () => {}}) => {
   const {ENABLE_DEV_ONLY_FEATURE} = configEnv;
-  const {setProfileId} = useProfileHook();
+  const {setAuth} = useUserAuthHook();
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `setAuth` function is being used instead of the previous `setProfileId`. Make sure that `setAuth` is properly updating the state and triggering necessary side effects (if any) as `setProfileId` did. Also, ensure that the state changes are handled correctly in other parts of the application that rely on this state.
```diff
- const {setProfileId} = useProfileHook();
+ const {setAuth} = useUserAuthHook();
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->